sdk/storage/storage-blob/src/BlobDownloadResponse.ts

```diff
@@ -471,6 +470,28 @@ export class BlobDownloadResponse implements BlobDownloadResponseModel {
     return this.originalResponse.contentCrc64;
   }
 
+  /**
+   * Object Replication Policy Id of the destination blob.
```

Comment:
Shall we align the comments with [https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob-properties](https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob-properties). 

source and destination blob look confused.